src/Tag/index.js

```diff
@@ -16,7 +16,7 @@ import styled from 'styled-components';
  */
 
 const Tag = ({ children, className, color, ...rest }) => (
-  <div className={className} {...rest}>
+  <div className={className} {...rest} data-testid="tag">
```

Comment:
Same question here for the Tag, why `data-testid` and not `data-test-id` ?

Reply:
@AntoninMarchardDev is part of the react-testing-library api